### PR TITLE
add anonymizer docs

### PR DESCRIPTION
**Pull request by shiranr**
Add anonymizer docs and how to add new anonymizer.

### CODE REVIEW

**Review by navalev — APPROVED** [review]

File: docs/anonymizer/index.md [hunk 1]
```diff
@@ -1,22 +1,194 @@
...
+  <PHONE_NUMBER\>
+- One PII is contained is another - Inigo was recognized as FIRST_NAME and Inigo Montoya
+  was recognized as NAME, we will take the larger one:
+  My name is <NAME\>. You Killed my Father. Prepare to die. BTW my number is: 03-232323.
+- Partial intersection - the number 03-2323 is recognized as a PHONE_NUMBER but 232323
+  is recognized as SSN:
+  My name is Inigo Montoya. You Killed my Father. Prepare to die. BTW my number is:
+  <PHONE_NUMBER\><SSN\>.
+
+=== "Python"
```
Comment (omri374) [inline docs/anonymizer/index.md:96]: Consider moving these under "Getting started"

File: docs/anonymizer/index.md [hunk 2]
```diff
@@ -1,22 +1,194 @@
...
+    docker build . -t presidio/presidio-anonymizer
+    ```
+
 ## Getting started
 
-- Running in Python.
-- Running using Docker.
-- Creating new anonymizers.
-- API reference.
+### Built-in anonymizers:
```
Comment (omri374) [inline docs/anonymizer/index.md:49]:
```suggestion
## Built-in anonymizers:
```

File: docs/anonymizer/index.md [hunk 3]
```diff
@@ -1,22 +1,194 @@
...
+| hash | hash the PII using either sha256, sha512 or md5 | `hash_type` - sets the type of hashing. Can be either `sha256`, `sha512` or `md5`. <br> The default hash type is `sha256`. | 
+| mask | replaces the PII with a given character | `chars_to_mask` - the amount of characters out of the PII that should be replaced. <br> `masking_char` - the character to be replaced with. <br> `from_end` - Whether to mask the PII from it's end. |
+
+!!! note "Note"
+    If anonymizers map is empty or "DEFAULT" key is not stated, the default
+    anonymizer is "replace" for all entities. The replacing value will be the entity type
+    e.g.: <PHONE_NUMBER\>
+
+
+### Overlapping Anonymization Scenarios
```
Comment (omri374) [inline docs/anonymizer/index.md:64]:
```suggestion
## Overlapping Anonymization Scenarios
```

File: docs/anonymizer/index.md [hunk 4]
```diff
@@ -1,22 +1,194 @@
...
+        {
+            "start": 48,
+            "end": 57,
+            "score": 0.95,
+            "entity_type": "PHONE_NUMBER"
+        }
+    ]}
+    ```
 
 ### Creating new anonymizers
```
Comment (omri374) [inline docs/anonymizer/index.md:183]:
```suggestion
## Creating new anonymizers
```

File: docs/anonymizer/index.md [hunk 5]
```diff
@@ -1,22 +1,194 @@
...
 
+Presidio anonymizer can be easily extended to support additional anonnymization methods.
+See [this tutorial on adding new anonymization methods](adding_anonymizers.md)
+for more information.
+
 ### API reference
```
Comment (omri374) [inline docs/anonymizer/index.md:189]:
```suggestion
## API reference
```

**Pull request merged**